From 2ab962bf23c9cc08db2e7a7d475e1ca60d5e08a0 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Fri, 17 Aug 2018 09:20:05 -0700
Subject: [PATCH 07/19] lineage: Always show option for swipe gesture nav bar

Change-Id: I243992335282456aa8198228382094ddeb5e577b

# Conflicts:
#	overlay/common/frameworks/base/core/res/res/values/config.xml
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index 9e5cb3a1..e6c1367f 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -92,4 +92,8 @@
 
     <!-- Flag indicating whether round icons should be parsed from the application manifest. -->
     <bool name="config_useRoundIcon">true</bool>
+
+    <!-- Whether or not swipe up gesture's opt-in setting is available on this device -->
+    <bool name="config_swipe_up_gesture_setting_available">true</bool>
+
 </resources>
-- 
2.17.1

